### ISSUE TITLE
Screen reader private vs public notebook differences + language icon changes

### ISSUE THREAD

**Issue by manfromjupyter** [nbgallery/nbgallery#825]
Private notebooks read:
"Notebook written in Python version 3.4.2 private notebook NOTEBOOK_TITLE"

Public notebooks read:
"Notebook written in Python version 3.4.2 NOTEBOOK_TITLE"

PROPOSED (get rid of the written in part and make sure public mentioned the title is a notebook's title.
"Python version 3.4.2 private notebook NOTEBOOK_TITLE"
&
"Python version 3.4.2 notebook NOTEBOOK_TITLE"

For Language icons. Despite using screen-reader-only text, the image needs alt text (just in case they ever fail to load or to at least trick the automated accessibility testing tools that they are fine.
